src/main/java/net/cytonic/cytosis/Manager.java

```diff
@@ -0,0 +1,36 @@
+package net.cytonic.cytosis;
+
+import net.cytonic.cytosis.data.Database;
+import net.cytonic.cytosis.logging.Logger;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+
+public class Manager {
```

Comment:
How about DatabaseManager?
Or DataManager, Or anything to reflect *what* it manages.